vogue/load/analysis.py

```diff
@@ -1,16 +1,24 @@
-from vogue.adapter import VougeAdapter
+import logging
+
+from genologics.entities import Sample
+from genologics.lims import Lims
+
 from vogue.build.analysis import build_analysis
 
-def load_cancer_analysis(adapter: VougeAdapter, lims_id=None, dry_run=False, analysis : dict={}):
+LOG = logging.getLogger(__name__)
+
+def load_analysis(adapter, dry_run=False, analysis : dict={}):
     """Load information from a cancer analysis"""
     
-    sample_obj = adapter.sample(lims_id)
-    if not sample_obj:
-        raise SyntaxError("Sample {} does not exist".format(lims_id))
-    
-    ## TODO build the analysis object
-    analysis_obj = build_analysis(analysis, 'QC' )
-    ## TODO load the object with adapter
-    adapter.load_analysis(analysis_obj)
-    
-    return analysis_obj
+    if dry_run:
+        existing_sample = adapter.sample(lims_sample.id)
```

Comment:
```suggestion
        existing_analysis = adapter.analysis(analysis['_id'])
```
Should be something like that? No?

Reply:
done!